api/default_networks.go

```diff
@@ -81,7 +81,7 @@ var sepolia = params.Network{
 var optimism = params.Network{
 	ChainID:                optimismChainID,
 	ChainName:              "Optimism",
-	RPCURL:                 "https://optimism-mainnet.rpc.grove.city/v1/",
+	RPCURL:                 "https://optimism-archival.rpc.grove.city/v1/",
```

Comment:
I know we needed client-side changes for changing RPC urls at some point. Is that no longer the case?